Sources/NerdzUtils/UI Components/LoadableButton.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  LoadableButton.swift
+//  
+//
+//  Created by Roman Kovalchuk on 28.04.2021.
+//
+
+import UIKit
+
+open class LoadableButton: UIButton {
+    
+    public var onStartLoading: (() -> Void)?
+    public var onFinishLoading: (() -> Void)?
+    // Activity indicator spacing, must be set before reasingning of activity indicator view
+    public var topBottomIndicatorPadding = CGFloat(10)
+
+    // Variable that represents loading state, change it if you want to show/hide loading indicator
+    public var isLoading: Bool = false {
+        didSet {
+            configureLoadingStateChange()
+        }
+    }
+    
+    // If you don't want to use default activity indicator, or want to customize it, you should set this variable
+    public var activityIndicatorView: UIView {
+        get {
+            indicatorView
+        }
+        
+        set {
+            if indicatorView.superview != nil {
```

Comment:
use guard

Reply:
It's needed to execute code in both cases, so can't change it to guard